tests/unit/utils/test_rsax931.py

```diff
@@ -108,3 +123,76 @@ def test_verifier(self):
 
         msg = verifier.verify(RSAX931Test.hello_world_sig)
         self.assertEqual(RSAX931Test.hello_world, msg)
+
+    def test_find_libcrypto(self):
```

Comment:
I'd prefer these be split up into separate tests, one for each operating system.

Reply:
Done.